pkgs/desktops/lomiri/default.nix

```diff
@@ -1,65 +1,68 @@
-{ lib
-, pkgs
-, libsForQt5
+{
+  lib,
+  pkgs,
+  libsForQt5,
 }:
 
 let
-  packages = self: let
-    inherit (self) callPackage;
-  in {
-    #### Core Apps
-    lomiri = callPackage ./applications/lomiri { };
-    lomiri-calculator-app = callPackage ./applications/lomiri-calculator-app { };
-    lomiri-camera-app = callPackage ./applications/lomiri-camera-app { };
-    lomiri-clock-app = callPackage ./applications/lomiri-clock-app { };
-    lomiri-filemanager-app = callPackage ./applications/lomiri-filemanager-app { };
-    lomiri-system-settings-unwrapped = callPackage ./applications/lomiri-system-settings { };
-    lomiri-system-settings-security-privacy = callPackage ./applications/lomiri-system-settings/plugins/lomiri-system-settings-security-privacy.nix { };
```

Comment:
Need aliase for package removed

Reply:
Added an alias for `lomiri-system-settings-security-privacy` -> `lomiri-system-settings-unwrapped`.